AR-Framework/src/at/fhv/itb6/arp/inputInterface/InputDetection.java

```diff
@@ -31,7 +34,7 @@ public InputDetection(InputConfiguration inputConfiguration){
      * @throws GamebordersNotDetectedException
      * @throws NoMarkerDetectedException
      */
-    public InputAction getUserInput() throws GamebordersNotDetectedException, NoMarkerDetectedException {
+    public InputAction getUserInput(CursorStatusListener callback) throws GamebordersNotDetectedException, NoMarkerDetectedException {
```

Comment:
Quick idea! are we not just geting all the 24 game board postions and then look if your cursor is on that position? then we could just use the enums that i implemented in the game facade to indicate to the game where the curser currently is? Just a quick thought!